commitlint.config.js

```diff
@@ -43,7 +43,6 @@ module.exports = {
         'dropdown',
         'e2e',
         'episode',
-        'favorite',
```

Comment:
@seferturan Not from the API definition though :trollface: 

Someone got overexcited 😂